objectserver/indexdb/ecobj.go

```diff
@@ -296,100 +299,158 @@ func (o *ecObject) Replicate(prirep objectserver.PriorityRepJob) error {
 	return err
 }
 
-func (o *ecObject) Stabilize(ring ring.Ring, dev *ring.Device, policy int) error {
-	wg := sync.WaitGroup{}
-	success := true
+func (o *ecObject) nurseryReplicate(rng ring.Ring, partition uint64, nodes []*ring.Device) error {
```

Comment:
Seems like this function should check if one of the nodes it's about to send to is itself.
Also, if it determines it is not a primary and it successfully sends the object out to the primaries, it should probably delete the local copy.